### PR TITLE
check if domains exist before redirecting

### PR DESCRIPTION
**Pull request by jurre**
Depends on https://github.com/DefactoSoftware/LearningSpaces/pull/834

### CODE REVIEW

**Comment by dkhgh** [comment]
Yes!

File: source/javascripts/sign-in.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+var tld = window.location.hostname.split('.').pop();
+
+$(function () {
+  $('#tld').text(".learningspaces." + tld);
+  var onClickHandler = function () {
+    var subdomain = $('#subdomain').val();
+    var endpoint = "https://learningspaces.io/api/v1/accounts/" + subdomain;
```
Comment (romkebon) [inline source/javascripts/sign-in.js]:
```javascript
var endpoint = "https://letsgo.learningspaces." + tld + "/api/v1/accounts/" + subdomain;
```
?

Comment (jurre) [inline source/javascripts/sign-in.js]: public right? But yeah!

**Comment by jurre** [comment]
@dkhgh I think this is good to go

**Comment by dkhgh** [comment]
@jurre is this part of our API on production now?

**Comment by jurre** [comment]
@dkhgh yes but i think i meed to enable CORS there since our landing page is not on https

**Pull request merged**